dags/public-health/covid19/jhu-to-esri.py

```diff
@@ -156,28 +116,16 @@ def load_jhu_global_time_series(branch="master"):
 
     # join
     df = df.assign(
-        number_of_deaths=deaths_df.deaths, number_of_recovered=recovered_df.recovered,
+        number_of_deaths=deaths_df.deaths,
+        number_of_recovered=recovered_df.recovered,
+        date=pd.to_datetime(df.date).dt.tz_localize("UTC"),
```

Comment:
I think that this will still show the timezone we don't want in the Esri frontends. If we always want to show the date that lines up in US/Pacific, I think we need to do the trick with `date` above.

Reply:
Do you mean `date = pd.Timestamp.now(tz="US/Pacific").normalize().tz_convert("UTC")`?